src/wp-includes/rest-api/endpoints/class-wp-rest-taxonomies-controller.php

```diff
@@ -272,16 +272,7 @@ public function prepare_item_for_response( $item, $request ) {
 		// Wrap the data in a response object.
 		$response = rest_ensure_response( $data );
 
-		$response->add_links(
-			array(
-				'collection'              => array(
-					'href' => rest_url( sprintf( '%s/%s', $this->namespace, $this->rest_base ) ),
-				),
-				'https://api.w.org/items' => array(
-					'href' => rest_url( rest_get_route_for_taxonomy_items( $taxonomy->name ) ),
-				),
-			)
-		);
+		$response->add_links( $this->prepare_links( $taxonomy ) );
```

Comment:
?

Reply:
This PR was branched from #2883, so it includes those changes. Once I split #2883 into separate PRs as requested, I'll update this PR.